src/player.js

```diff
@@ -390,6 +390,12 @@ export default class Player extends FakeEventTarget {
    * @private
    */
   _resizeWatcher: ResizeWatcher;
+  /**
+   * Holds preset component factories
+   * @type {?PresetComponent}
+   * @private
+   */
+  _presetComponents: PresetComponent[];
```

Comment:
```suggestion
  _presetComponents: Array<PresetComponent>;
```
fix the the jsdoc accordingally